lms/static/scripts/frontend_apps/components/ContentSelector.js

```diff
@@ -147,28 +147,14 @@ export default function ContentSelector({
     case 'url':
       dialog = <URLPicker onCancel={cancelDialog} onSelectURL={selectURL} />;
       break;
-    case 'canvasFile':
+    case 'lmsFile':
```

Comment:
Can you get most of what you want by changing this to a default case or a list of values?